doc/api/process.md

```diff
@@ -2593,6 +2593,57 @@ the [`'warning'` event][process_warning] and the
 [`emitWarning()` method][process_emit_warning] for more information about this
 flag's behavior.
 
+## `process.parseArgs`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+* `config` {Object} Used to provide arguments for parsing and to configure
+  the parser. `config` supports the following properties:
+  * `args` {string\[]} array of argument strings. **Default:** `process.argv`
+    with `execPath` and `filename` removed.
+  * `options` {Object} Used to describe arguments known to the parser.
+    Keys of `options` are the long names of options and values are an
+    {Object} accepting the following properties:
+    * `type` {string} Type of argument, which must be either 'boolean' or `string`.
+      **Default:** `boolean`.
+    * `multiple` {boolean} Should the option accept multiple arguments.
+    * `short` {string} A single character alias for the option.
+  * `strict`: {boolean} Should an error be thrown if unknown options are
+    encountered.
+    **Default:** `false`.
+* Returns: {Object} An {Object} representing the parsed command line
+  arguments:
+  * `flags` {Object} With properties and {boolean} values corresponding to
+    parsed options passed.
+  * `values` {Object} With properties and {string} values corresponding to
+    parsed options passed.
+  * `positionals` {string\[]}, containing positional arguments.
+
+Provides a higher level API for command-line argument parsing than interacting
+with `process.argv` directly.
+
+```mjs
+import { parseAargs } from 'process';
```

Comment:
```suggestion
import { parseArgs } from 'process';
```

Reply:
I'm moving the docs around based on feedback, making these changes as I do so.